app/src/main/java/com/vitorpamplona/amethyst/model/HashtagIcon.kt

```diff
@@ -0,0 +1,15 @@
+package com.vitorpamplona.amethyst.model
+
+import androidx.compose.ui.graphics.Color
+import com.vitorpamplona.amethyst.R
+fun checkForHashtagWithIcon(tag: String): HashtagIcon? {
+    if (tag.lowercase() == "bitcoin" || tag.lowercase() == "btc") {
```

Comment:
Does this change the #bitcoin hashtag to #btc?

Reply:
No, it adds a Bitcoin Icon to Bitcoin, bitcoin, BTC and btc hashtags (Like it's done in Twitter)

Reply:
Oh got it, didn't read it's the description not the tag text. Cool.